README.md

```diff
@@ -1,28 +1,30 @@
 # Slice Extension for Visual Studio Code
 
-This simple [language extension](https://code.visualstudio.com/api/language-extensions/overview) adds support for [Slice](https://doc.zeroc.com/ice/latest/the-slice-language) syntax highlighting to Visual Studio Code and other environments, such as GitHub and the Visual Studio IDE.
+This language extension adds support for highlighting Slice files within Visual Studio Code.  
+It supports `.ice` files (which use the [original Slice](https://doc.zeroc.com/ice/latest/the-slice-language) syntax)
+and `.slice` files (which use the [latest Slice](https://docs.icerpc.dev/slice2) syntax).
 
 ## Installation
 
-### Visual Studio Code
-
-The [Slice Highlighter](https://marketplace.visualstudio.com/items?itemName=ZeroCInc.slice) extension can be installed from the Visual Studio Code Marketplace or through VS Code Quick Open (`Ctrl + P`) with the following command:
+The extension can be installed through the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ZeroCInc.slice) by clicking "Install",  
+or from within VSCode by using Quick Open (`Ctrl + P`) to run the following command:
 ```
 ext install "Slice Highlighter"
 ```
 
-To manually install this extension from source, first navigate to the `.vscode/extensions` folder in your home directory (If this folder doesn't already exist, you'll have to create it), and copy this repository into it: `.vscode/extensions/vscode-slice`.
-```
-cd ~/.vscode/extensions
-git clone https://github.com/zeroc-ice/vscode-slice.git
-```
```

Comment:
I seen some extensions have a "manual" install option. I agree that it should be oriented towards VSCode and doing things the easy way. What seems to be the consensus now for popular extensions about keeping manual install instructions?

Reply:
*shrugs* I don't know, but I can check. I suspect it's the wild west though.

Even if we don't explicitly say "manual installation" anymore, note that you can follow the instructions in the "developement" section to locally package and install the extension, which is basically the same thing.

Reply:
Was just looking into it, does seem like the Wild West. I would defo get rid of it though, many big ones like the tailwind css extension don't include it.

The only thing I see quite commonly is a configuration section but we don't support any configurations.

Reply:
Same story with the Rust and Protobuf ones.

There's definitely other things that are customary to include like any commands it adds, or configuration like you point out.
But those won't matter until we expand the extension to include a proper language server!